packages/apps/lightning-ui-docs/.storybook/addons/toolbars/ThemeDownload.js

```diff
@@ -17,10 +17,10 @@
  */
 
 import React from 'react';
+import { DOWNLOAD_ID } from '../constants';
 import { Icons, IconButton } from '@storybook/components';
 import { convertNumToHexAlphaArray } from '../../utils/helpers';
-import { utils } from '@lightningjs/ui-components';
-import { DOWNLOAD_ID } from '../constants';
+import { utils } from '@lightningjs/ui-components/src';
```

Comment:
same as the withLightning file, why is this `/src` needed? Aren't utils exported by ui-components already?

Reply:
Gotcha. Thanks for the explanation! Hopefully some future version will "fix" that then 😅